### PR TITLE
added require files

### PR DESCRIPTION
**Pull request by madhu-lemosys**

### CODE REVIEW

File: WRIO.js [hunk 1]
```diff
@@ -9,13 +9,47 @@
  * ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
  * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
-     
-var importUrl = 'https://raw.githubusercontent.com/webRunes/';
```
Comment (AlexeyAnshakov) [inline WRIO.js:13]: Why this was deleted?

File: WRIO.js [hunk 2]
```diff
@@ -242,13 +315,29 @@ var CreateArticle = React.createClass({
...
+      <div className="commentList">
+        {commentNodes}
+      </div>
+    );*/
   }
 });
 
 var CreatArticleEl = React.createClass({
   loadArticleFromServer: function(title,children,hasPart,rawMarkup) {
-	var url = importUrl + 'Default-WRIO-Theme/gh-pages/widget/article.htm';
```
Comment (AlexeyAnshakov) [inline WRIO.js:251]: Restore it

File: WRIO.js [hunk 3]
```diff
@@ -169,7 +203,32 @@ var CreateTitter = React.createClass({
         <CreateOneTitter data={this.state.data} />
     );	
   }
-
+  /*
```
Comment (AlexeyAnshakov) [inline WRIO.js:206]: Why it there? Titter markup have to be taken from 
http://wrio.s3-website-us-east-1.amazonaws.com/Titter-WRIO-App/widget/titter.htm

**Comment by madhu-lemosys** [comment]
changes in wrio

**Comment by AlexeyAnshakov** [comment]
https://github.com/webRunes/WRIO-InternetOS/issues/55

**Comment by AlexeyAnshakov** [comment]
https://github.com/webRunes/WRIO-InternetOS/issues/29

**Pull request merged**